src/components/ChatLog.jsx

```diff
@@ -0,0 +1,28 @@
+import * as PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.jsx';
+
+const ChatLog = ({ entries, onToggleLike }) => {
+  return (
+    <div>
+      {entries.map((entry) => (
+        <ChatEntry
+          key={entry.id}
+          id={entry.id}
+          body={entry.body}
+          sender={entry.sender}
+          timeStamp={entry.timeStamp}
+          liked={entry.liked}
+          onToggleLike={onToggleLike}
+          isLocal={entry.sender === 'Vladimir'}
+        />
+      ))}
+    </div>
+  );
+};
```

Comment:
Typically you'll see the result of using `map` referenced by a variable and using that variable in the JSX, like:

```js
  const chatLogs =  entries.map((entry) => {
	      return (
	          <ChatEntry                                 
	            key={entry.id}
                    {...entry}
                    onLikeToggle={onLikeToggle}
                    isLocal={entry.sender === 'Vladimir'}
	          />
	        );
	     });
  };

  return (
        <div>
            {chatLogs}
        </div>
  );
```